src/building/how-to-build-and-run.md

```diff
@@ -212,6 +212,11 @@ host: x86_64-unknown-linux-gnu
 release: 1.48.0-dev
 LLVM version: 11.0
 ```
+
+The rustup toolchain points to the specified toolchain compiled in your `build` directory,
+so the rustup toolchain will be updated whenever `x.py build` or `x.py test` are run for
+that toolchain/stage.
```

Comment:
Hmm, do you think this wasn't clear before? It's definitely true but this section is already kind of long.

Reply:
It wasnt clear to me so I had to test it out to verify how it worked.

I think in particular it might be surprising that `x.py test` can affect the rustup toolchain.
I thought that this wasnt the case for cargo, but I just tested it and `cargo test` does overwrite the output of `cargo build`!

So if its too long I could shorten it to:
```
Note that both `x.py build` and `x.py test` will update the linked rustup toolchain.
```

Otherwise if this is all obvious then im happy to remove it entirely.
I totally agree that saying obvious things is wasting everyone's bandwidth.

Reply:
> It wasnt clear to me so I had to test it out to verify how it worked.

👍 Seems good to add this bit then.